src/ecephys_etl/modules/align_timestamps/channel_states.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+
+from . import barcode
+
+
+def extract_barcodes_from_states(
+    channel_states, timestamps, sampling_rate, **barcode_kwargs
+):
+    """Obtain barcodes from timestamped rising/falling edges.
+
+    Parameters
+    ----------
+    channel_states : numpy.ndarray
+        Rising and falling edges, denoted 1 and -1
+    timestamps : numpy.ndarray
+        Sample index of each event.
+    sampling_rate : numeric
+        Samples / second
+    **barcode_kwargs : 
```

Comment:
Where are these additional parameters defined? All of this passing around of `**barcode_kwargs` makes me uneasy. Is there no way to explicitly define the expected arguments (or have some kind of parameter container class that can carry them around?)